EIPS/eip-eof-metadata.md

```diff
@@ -0,0 +1,88 @@
+---
+eip: xxx
+title: Separate Metadata Section for EOF
+description: Introduce a new separate metadata section to the EOF
+author: Kaan Uzdogan (@kuzdogan), Marco Castignoli (@marcocastignoli), Manuel Wedler (@manuelwedler)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5

Reply:
Added